src/chart.js

```diff
@@ -105,6 +104,15 @@ Chart.canvasHelpers = Chart.helpers.canvas;
  */
 Chart.layoutService = Chart.layouts;
 
+/**
+ * Provided for backward compatibility, not available anymore.
```

Comment:
Provide a `what to use instead` in comment, like other backward compatibility items do?

Reply:
There is no alternative other than re-implementing the linear scale logic from scratch. Not sure yet what we will do in v3, but @etimberg is right, this base scale shouldn't have been public.